### PR TITLE
fix: socket.connnect JS : Scope & Closures

### PR DESCRIPTION
**Pull request by mojinxun**
when io reconnect, a new socket well create.
but from front socket not be updated

the commit is fix it by global.elesocket

please check, or better method

### CODE REVIEW

File: ElectronNET.CLI/ElectronNET.CLI.csproj [hunk 1]
```diff
@@ -5,7 +5,6 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>dotnet-electronize</AssemblyName>
     
-    <PackageType>DotnetCliTool</PackageType>
```
Comment (robertmuehsig) [inline ElectronNET.CLI/ElectronNET.CLI.csproj]: Be aware: This is needed and should not be deleted!

File: ElectronNET.CLI/ElectronNET.CLI.csproj [hunk 2]
```diff
@@ -17,7 +16,7 @@
 This package contains the dotnet tooling to electronize your application.</Description>
     <PackageLicenseUrl>https://github.com/ElectronNET/Electron.NET/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/ElectronNET/Electron.NET/</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/ElectronNET/Electron.NET/</RepositoryUrl>
```
Comment (robertmuehsig) [inline ElectronNET.CLI/ElectronNET.CLI.csproj]: Shouldn't be deleted as well

**Comment by robertmuehsig** [comment]
Hi @mojinxun 
Your changes seems to touch some non related files. Could you cleanup this PR or open a new one with only the needed changes from you?
Thanks!

**Comment by mojinxun** [comment]
ok， I create nuget, something error, so I delete it

**Comment by mojinxun** [comment]
I update it, please review it

**Comment by robertmuehsig** [comment]
@GregorBiswanger Please take a look.

**Comment by robertmuehsig** [comment]
@GregorBiswanger What about this PR?

**Pull request merged**